index.html

```diff
@@ -55,6 +55,157 @@ <h6>I'am a Software Developer</h6>
+                        </ul>
+                    </div>
+                    <button class="button-p" type="button"><span class="see-project">See Project</span></button>
+                </div>
+                <div class="project-4">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div>
+                        <ul class="tags">
+                            <li class="tag-p1">html</li>
+                            <li class="tag-p2">bootstrap</li>
+                            <li class="tag-p3">Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button-p" type="button"><span class="see-project">See Project</span></button>
+                </div>
+                <div class="project-5">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div>
+                        <ul class="tags">
+                            <li class="tag-p1">html</li>
+                            <li class="tag-p2">bootstrap</li>
+                            <li class="tag-p3">Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button-p" type="button"><span class="see-project">See Project</span></button>
+                </div>
+                <div class="project-6">
+                    <h4>Profesional Art Printing Data</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <div>
+                        <ul class="tags">
+                            <li class="tag-p1">html</li>
+                            <li class="tag-p2">bootstrap</li>
+                            <li class="tag-p3">Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button-p" type="button"><span class="see-project">See Project</span></button>
+                </div>
+            </div>
+        </div>
+        <div class="about-myself-section">
+            <div>
+                <h1 class="about-me">About Me</h1>
+                <p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some
+                    of my work and experience! If you like what you see and have a project you need coded, don’t
+                    hestiate to contact me.</p>
+                <button class="get-resume" type="button">
+                    <span class="gr-btn-txt">
+                        Get My Resume
```

Comment:
Also here you should remove the border and implement the effects on hover and active for more details check the comment below

Reply:
done

Reply:
done